Zend/tests/fibers/suspend-outside-fiber.phpt

```diff
@@ -7,7 +7,7 @@ $value = Fiber::suspend(1);
 
 ?>
 --EXPECTF--
-Fatal error: Uncaught FiberError: Cannot suspend outside of a fiber in %ssuspend-outside-fiber.php:%d
```

Comment:
I think in this particular instance the old description is more obvious than "nowhere to go".